protocol/communities/manager.go

```diff
@@ -4897,18 +4897,22 @@ func (m *Manager) ShareRequestsToJoinWithPrivilegedMembers(community *Community,
 	return nil
 }
 
-func (m *Manager) shareAcceptedRequestToJoinWithPrivilegedMembers(community *Community, requestsToJoin *RequestToJoin) error {
-	pk, err := common.HexToPubkey(requestsToJoin.PublicKey)
+func (m *Manager) shareAcceptedRequestToJoinWithPrivilegedMembers(community *Community, requestToJoin *RequestToJoin) error {
```

Comment:
```suggestion
func (m *Manager) shareAcceptedRequestToJoinWithPrivilegedMembers(community *Community, requestToJoin *const RequestToJoin) error {
```
@ google can you please introduce this concept?